apiserver/cmd/apiserver/server/options.go

```diff
@@ -102,6 +102,9 @@ func (o *CalicoServerOptions) Config() (*apiserver.Config, error) {
 		}
 	}
 
+	if err := o.RecommendedOptions.Etcd.Complete(serverConfig.StorageObjectCountTracker, serverConfig.DrainedNotify(), serverConfig.AddPostStartHook); err != nil {
```

Comment:
Complete must be called exactly once before using any of the Apply methods. This change is similar to the upstream kubernetes/apiserver [implementation](https://github.com/kubernetes/kubernetes/blob/release-1.26/staging/src/k8s.io/apiserver/pkg/server/options/recommended.go#L104).